defaults/main.yml

```diff
@@ -13,9 +13,13 @@ karaf_log_path: ${karaf.data}/log/
 
 karaf_logging_template: org.ops4j.pax.logging.cfg
 
-karaf_java_home: /usr/lib/jvm/java-8-openjdk-amd64
+karaf_java_home: "{% if ansible_os_family == 'RedHat' %}/usr/lib/jvm/java-1.8.0-openjdk/{% else %}/usr/lib/jvm/java-8-openjdk-amd64{% endif %}"
 
-karaf_java_packages:
+karaf_java_packages_yum:
```

Comment:
I'd like to see this remain `karaf_java_pacakges` and then set it to the appropriate value for cent or for ubuntu, same with `karaf_java_home`. 

I'm a fan of that way [this role](https://github.com/geerlingguy/ansible-role-mysql) handles OS specific settings, so they can be overridden in a playbook and they can get sensible default values. 

Maybe something like this:
https://github.com/geerlingguy/ansible-role-mysql/blob/master/tasks/variables.yml

Reply:
Testing an alternative now.

Reply:
I should note that [my recent Grok PR](https://github.com/Islandora-Devops/ansible-role-grok/pull/2) avoided this by simply listing the packages in repo-specific tasks. Should I go back and revise it to be more like the I just committed?